pkg/registry/core/shoot/strategy.go

```diff
@@ -93,6 +93,11 @@ func mustIncreaseGeneration(oldShoot, newShoot *core.Shoot) bool {
 		return true
 	}
 
+	// Force delete annotation is set.
```

Comment:
Extend the comment and explain why it's necessary (it's not obvious right away).